frontend/js/src/album/AlbumPage.tsx

```diff
@@ -260,6 +260,7 @@ export default function AlbumPage(): JSX.Element {
       </div>
     );
   }
+  const artistForRadio = artist.artists?.[0]?.artist_mbid ?? encodeURIComponent(artistName);
```

Comment:
Instead of playing artist radio for just the first artist, we shuold send all artist uuids to the LB Radio to get the best results